packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.component.js

```diff
@@ -230,19 +231,22 @@ export class CheckoutOrderSummary extends PureComponent {
         const {
             totals: {
                 tax_amount,
+                shipping_tax_amount,
                 quote_currency_code,
                 cart_display_config: {
                     include_tax_in_order_total
                 } = {}
-            }
+            },
+            checkoutStep
         } = this.props;
         const title = __('Order total');
         const orderTotal = this.getOrderTotal();
+        const shippingTax = checkoutStep === SHIPPING_STEP ? shipping_tax_amount : 0;
```

Comment:
What if some project has more than shipping -> billing -> details steps? Maybe it's better to check for BILLING_STEP?